packages/experimental-nextjs-app-support/package-shape.json

```diff
@@ -42,10 +42,18 @@
   },
   "@apollo/experimental-nextjs-app-support/ssr": {
```

Comment:
The `/ssr` and `/rsc` exports have been deprecated for a while and I've removed them from the new package.
The fallback "redirect" will now handle all environments the same, and forward all of these to the new package, where they might be `undefined`, but present, depending on the environment.
It's a compromise to make things not too complicated.